### PR TITLE
ifstat: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
This is a very lazy starting point. If you know more, please add them

File: pages/linux/ifstat.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ifstat
+
+> View network interface statistics.
+> More information: <https://github.com/matttbe/ifstat?tab=readme-ov-file>.
```
Comment (spageektti) [inline pages/linux/ifstat.md:4]:
```suggestion
> More information: <https://github.com/matttbe/ifstat>.
```

**Comment by Managor** [comment]
Maybe later. Currently going through https://linuxcommandlibrary.com/basic/oneliners

**Comment by Managor** [comment]
Done.

**Review by spageektti — APPROVED** [review]

**Pull request merged**